packages/react-components/react-data-grid-react-window/src/components/DataGridBody/useDataGridBody.ts

```diff
@@ -12,14 +15,17 @@ import { useDataGridBody_unstable as useDataGridBodyBase_unstable } from '@fluen
  * @param ref - reference to root HTMLElement of DataGridBody
  */
 export const useDataGridBody_unstable = (props: DataGridBodyProps, ref: React.Ref<HTMLElement>): DataGridBodyState => {
-  const { height, itemSize, width = '100%' } = props;
-  const baseState = useDataGridBodyBase_unstable(props, ref);
+  const { height, itemSize, width = '100%', children } = props;
+
+  // cast the row render function to work with unknown args
+  const renderRowWithUnknown = children as (row: TableRowData<unknown>, ...rest: unknown[]) => React.ReactNode;
```

Comment:
```suggestion
  const renderRowWithUnknown = children as RowRenderFunction;
```

There is `RowRenderFunction` that defined it. Is it exported?